pkg/config/config.go

```diff
@@ -486,6 +499,24 @@ func checkBackend(bep backendPath, be TerraformBackend) error {
 	return nil
 }
 
+func checkProviders(pp providerPath, tp map[string]TerraformProviders) error {
+	for k, v := range tp {
+		if v.Source == "" {
+			return BpError{pp.Type, errors.New(fmt.Sprintf("provider, %s, is missing source", k))}
+		}
+		if v.Version == "" {
+			return BpError{pp.Type, errors.New(fmt.Sprintf("provider, %s, is missing version", k))}
+		}
+		req_ps := [...]string{"project", "region", "zone"}
```

Comment:
I think it's a stretch to assume what fields are required.

Reply:
Took out these checks and adjusted tests.  Done.